proto/enums/enums.proto

```diff
@@ -333,6 +333,23 @@ enum StatusCode {
     TEST_CST_USER_ERR = 1102;
     // Unable to docker pull image
     TEST_IMG_PULL_ERR = 1103;
+    // Error running custom test
+    TEST_CT_USER_ERR = 1104;
```

Comment:
can you expand `CT` to spell `CUSTOM` everywhere? I don't think the abbrev. is that clear.

Reply:
Updated.